ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/Constants.java

```diff
@@ -354,4 +354,7 @@ public String toString() {
     public static final String EXAMPLES = "examples";
     public static final String OPERATION_ID = "operationId";
     public static final String RESPONSE_ATTRIBUTE = "response";
+    public static final String REQUEST_ATTRIBUTE = "requestBody";
```

Comment:
Shouldn't this be
```suggestion
    public static final String REQUEST_BODY_ATTRIBUTE = "requestBody";
```
?